packages/beeq/src/components/page-title/bq-page-title.tsx

```diff
@@ -125,7 +125,7 @@ export class BqPageTitle {
         {/* Divider */}
         <div part="divider">
           <slot name="divider">
-            <bq-divider class="mb-m block" stroke-color="ui--secondary" stroke-thickness="1" />
+            <bq-divider class="block m-be-m" stroke-color="ui--secondary" stroke-thickness="1" />
```

Comment:
To properly support writing-mode, I think we should remove the bq-divider and use a `border-block-end` instead 🤔 

![CleanShot 2024-06-25 at 18 22 35](https://github.com/Endava/BEEQ/assets/328492/cabdfaae-5723-4eca-a55d-ac89ef6d833f)

The divider uses an SVG under the hood that won't adapt if the writing-mode changes.